src/plugins/intel_gpu/src/graph/impls/registry/lstm_seq_impls.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "primitive_inst.h"
+#include "registry.hpp"
+#include "intel_gpu/primitives/rnn.hpp"
+
+
+#if OV_GPU_WITH_ONEDNN
+    #include "impls/onednn/lstm_seq_onednn.hpp"
+#endif
+
+namespace ov {
+namespace intel_gpu {
+
+using namespace cldnn;
+
+const std::vector<std::shared_ptr<cldnn::ImplementationManager>>& Registry<lstm_seq>::get_implementations() {
+    static const std::vector<std::shared_ptr<ImplementationManager>> impls = {
+        OV_GPU_CREATE_INSTANCE_ONEDNN(onednn::LSTMSeqImplementationManager, shape_types::static_shape)
+        OV_GPU_GET_INSTANCE_OCL(lstm_seq, shape_types::static_shape)
```

Comment:
Please add impl manager for OCL version too.

Reply:
done